cmd/minishift/cmd/openshift/service.go

```diff
@@ -63,31 +51,23 @@ var serviceCmd = &cobra.Command{
 		api := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
 		defer api.Close()
 
-		url, err := cluster.GetServiceURL(api, namespace, service, serviceURLTemplate)
+		url, err := openshift.GetServiceURL(service, namespace)
 		if err != nil {
 			fmt.Fprintln(os.Stderr, err)
-			if _, ok := err.(cluster.MissingNodePortError); !ok {
-				fmt.Fprintln(os.Stderr, "Verify that Minishift is running and that the correct namespace is specified in the -n option.")
-			}
 			atexit.Exit(1)
 		}
 
-		if https {
-			url = strings.Replace(url, "http", "https", 1)
-		}
 		if serviceURLMode {
 			fmt.Fprintln(os.Stdout, url)
 		} else {
 			fmt.Fprintln(os.Stdout, "Opening the service "+namespace+"/"+service+" in the default browser...")
-			browser.OpenURL(url)
+			browser.OpenURL("http://" + url)
 		}
 	},
 }
 
 func init() {
-	serviceCmd.Flags().StringVarP(&namespace, "namespace", "n", "default", "The namespace of the service.")
+	serviceCmd.Flags().StringVarP(&namespace, "namespace", "n", "", "The namespace of the service.")
 	serviceCmd.Flags().BoolVar(&serviceURLMode, "url", false, "Access the service in the command-line console instead of the default browser.")
-	serviceCmd.PersistentFlags().StringVar(&serviceURLFormat, "format", "http://{{.IP}}:{{.Port}}", "The URL format of the service.")
-	serviceCmd.Flags().BoolVar(&https, "https", false, "Access the service with HTTPS instead of HTTP.")
```

Comment:
We agreed here https://github.com/minishift/minishift/issues/342#issuecomment-286039062 to drop these flags since these doesn't make sense at the moment to our use case.